roles/satellite-clone/tasks/restore.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Restore mongo data
   command: "tar --selinux --overwrite -xf {{ backup_dir }}/mongo_data.tar.gz -C /"
-  when: standard_backup
+  when: rhel_migration or standard_backup
```

Comment:
@sthirugn ok lets keep it scenario dependent and fix that  "only one scenario exists" check in a separate PR